packages/block-library/src/table/style.scss

```diff
@@ -34,8 +34,7 @@
 
 		td,
 		th {
-			// Aligned tables shouldn't scroll horizontally so we need their contents to wrap.
-			word-break: break-all;
+			overflow-wrap: break-word;
```

Comment:
Looks like you might have missed this line when updating to use word-break: break-word in your last commit?

Reply:
Arrrrggghghhh!!! Nice catch. Should be resolved now.